docs/src/documentation/03-components/05-input/inputfield/03-accessibility.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/inputfield/accessibility/
+---
+
+### Accessibility
+
+- For special cases you can use your own, detached `label`. Simply like this:
```

Comment:
I've just copied the info from README file. Will be polished in the https://kiwicom.atlassian.net/browse/FEPLT-2345